Formula/wsk.rb

```diff
@@ -1,6 +1,6 @@
 class Wsk < Formula
   desc "OpenWhisk Command-Line Interface (CLI)"
-  homepage "http://openwhisk.org"
+  homepage "https://openwhisk.incubator.apache.org/"
```

Comment:
This seems to be available on https://openwhisk.apache.org/ as well - is there a preference here ?